src/containerapp/azext_containerapp/custom.py

```diff
@@ -1083,7 +1083,9 @@ def containerapp_up(cmd,
     env = ContainerAppEnvironment(cmd, environment, resource_group, location=location, logs_key=logs_key, logs_customer_id=logs_customer_id, custom_location_id=custom_location_id, connected_cluster_id=connected_cluster_id)
     app = ContainerApp(cmd, name, resource_group, None, image, env, target_port, registry_server, registry_user, registry_pass, env_vars, workload_profile_name, ingress)
 
-    _set_up_defaults(cmd, name, resource_group_name, logs_customer_id, location, resource_group, env, app, custom_location, extension)
+    # Check and see if registry username and passwords are specified. If so, set is_registry_server_params_set to True to use those creds.
+    is_registry_server_params_set = bool(registry_server and registry_user and registry_pass)
```

Comment:
What if the cx using managed identity to pull ACR image?

Reply:
Sorry, I noticed the command `az containerapp up` parameters not support `--registry-identity`, which supported in command `az containerapp create`.